src/routes.js

```diff
@@ -0,0 +1,131 @@
+import 'babel-core/register';
```

Comment:
@gabro usually imports like this one live better in the entry file (aka `index.js`). Why did you leave them here?